.setup/ansible/playbooks/submitty_install.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Example playbook
+# Customize the submitty install by overriding the variable defaults found in `roles/submitty_install/defaults/main.yml`
+
+- name: Install Submitty
+  hosts: submitty
+  remote_user: ubuntu
+  become: yes
+  tasks:
+    - name: Install Submitty
+      ansible.builtin.include_role:
+        name: submitty_install
+      vars:
+        # add overriding variables here
+        submitty_version: v24.02.00
```

Comment:
How is this going to be kept up-to-date?  I see that the version reported in `.setup/ansible/roles/submitty_install/defaults/main.yml` is different.  Ensuring that they stay in sync seems like a good thing to test, either in this PR or in the upcoming tests PR.

Reply:
I agree. I updated the code to align submitty versions and added a reminder in the documentation. I will look at creating a test to ensure this.